app/controllers/requests_controller.rb

```diff
@@ -35,6 +39,20 @@ def destroy
 
   private
 
+  def total_items(requests)
+    request_items = []
+
+    requests.pluck(:request_items).each do |items|
+      items.map { |json| request_items << [Item.find(json['item_id']).name, json['quantity']] }
+    end
```

Comment:
I would have implemented this to avoid an N+1 query through the looping `Item.find`. Instead, I would fetch all the `Item` records and then retrieve them in this loop.

Another consideration is that in case an Item with the `item_id` could not be found (in case of bad data), it would cause the page to raise an error and return a 500.

The N+1 isn't going to be a big problem since our traffic is relatively low but I'll open an issue to address it.